README.md

```diff
@@ -41,6 +41,9 @@ type Options struct {
 	// OnDirExists can specify what to do when there is a directory already existing in destination.
 	OnDirExists func(src, dest string) DirExistsAction
 
+	// OnErr lets called decide whether or not to continue on particular copy error.
+	OnErr func(err error) error
```

Comment:
IMO, I guess giving `src` and `dest` is more friendly for users, which is like:

```go
OnError func(err error, src, dest string) error
```

I'd like to hear your opinion.